subsys/net/lib/lwm2m/lwm2m_engine.c

```diff
@@ -3363,6 +3363,150 @@ static int bootstrap_delete(struct lwm2m_message *msg)
 
 	return ret;
 }
+
+static inline int bs_discover_fill_enabler(struct lwm2m_message *msg)
+{
+	char buf[16];
+
+	snprintk(buf, sizeof(buf), "lwm2m=\"%s\"", LWM2M_PROTOCOL_VERSION);
+	return buf_append(CPKT_BUF_WRITE(msg->out.out_cpkt), buf, strlen(buf));
+}
+
+static inline int bs_discover_fill_object(struct lwm2m_engine_obj *obj,
+					  struct lwm2m_message *msg)
+{
+	char buf[16];
```

Comment:
similar comment here, so something like `char buf[sizeof(",</xxxxx>")];` depending on the size of obj_id